src/EditorFeatures/CSharpTest/EmbeddedLanguages/Json/CSharpJsonParserTests_TestGeneration.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if false
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.Json;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.EmbeddedLanguages.Json
+{
+    public class Fixture : IDisposable
+    {
+        public void Dispose()
+        {
+#if false
+            var other = new Dictionary<string, string>();
+
+            var tree = SyntaxFactory.ParseSyntaxTree(
+                File.ReadAllText(@"C:\GitHub\roslyn-internal\Open\src\Workspaces\CSharpTest\Json\CSharpJsonParserTests_BasicTests.cs"));
+
+            var methodNames = tree.GetRoot().DescendantNodes().OfType<MethodDeclarationSyntax>().Select(m => m.Identifier.ValueText);
+            var nameToIndex = new Dictionary<string, int>();
+
+            var index = 0;
+            foreach (var name in methodNames)
+            {
+                nameToIndex[name] = index;
+                index++;
+            }
+
+#if true
+            var tests =
+                CSharpJsonParserTests.nameToTest.Where(kvp => !kvp.Key.StartsWith("NegativeTest") && !kvp.Key.StartsWith("Reference"))
+                     .OrderBy(kvp => nameToIndex[kvp.Key])
+                     .Select(kvp => kvp.Value);
+#elif false
```

Comment:
?

Reply:
i'll remove all this.  however, for clarity, this was a useful way to make bulk baseline changes (like when i was changing the tree shape).  You would just run all the tests again, but this time the test inputs would go into a helper which would generate all the updated expected outputs (tree and diagnostics).  Then you could bulk overwrite them.